### PR TITLE
Add function for building proof object

### PR DESCRIPTION
**Pull request by fanatid**
Sorry, but I broke API for build/extract.

### CODE REVIEW

**Comment by mappum** [comment]
Thanks for adding this! :+1:

I've left some comments, if you address them I'll pull this.

**Comment by fanatid** [comment]
updated
I'm answered on your comments for previous commit, I don't see them here, [link](https://github.com/fanatid/bitcoin-merkle-tree/commit/240db0d65e0eea5e88ab811137b1259bd48c25ce)

**Comment by mappum** [comment]
Awesome, thanks. :smile:

File: package.json [hunk 1]
```diff
@@ -2,22 +2,28 @@
...
     "bitcoin",
     "merkle",
-    "tree"
+    "tree",
+    "bip37"
   ],
   "author": "Matt Bell <mappum@gmail.com>",
-  "license": "MIT",
-  "dependencies": {
-    "create-hash": "^1.1.2"
```
Comment (fanatid) [inline package.json:17]: @mappum sorry, can you return `create-hash` back?

**Pull request merged**